android/sync-crypto/build.gradle

```
@@ -0,0 +1,92 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+    id "com.vanniktech.maven.publish" version "0.23.1"
+}
+
+def getVersionName = { ->
+    def props = new Properties()
+    file("../version.properties").withInputStream { props.load(it) }
+    return props.getProperty("VERSION_NAME")
+}
+
+def buildVersionCode = {
+    def versionName = getVersionName()
+    def (major, minor, patch) = versionName.toLowerCase().tokenize('-')[0].tokenize('.')
+    (major, minor, patch) = [major, minor, patch].collect { it.toInteger() }
+    return (major * 10_000_000) + (minor * 10_000) + (patch * 1_000)
+}
+
+import com.vanniktech.maven.publish.SonatypeHost
+
+mavenPublishing {
+    // publishing to https://s01.oss.sonatype.org
+    publishToMavenCentral(SonatypeHost.S01, true)
+
+    signAllPublications()
+
+    coordinates("com.duckduckgo.ddgsynccrypto", "ddg-android-sync-crypto", "${getVersionName}")
```

As discussed
```suggestion
    coordinates("com.duckduckgo.cryptosync", "cryptosync-android", "${getVersionName}")
```

Named it `cryptosync` instead of `synccrypto` to avoid the `cc` :D but have no strong opinions about that

---

I was about to introduce the change, but I prefer consistency and there are lot of places where we refer to this as sync crypto (e.g: this repo, kotlin class, package on main repo, etc.)

Coordinates will be: `com.duckduckgo.synccrypto:sync-crypto-android:Major-minor-patch`